build_tools/benchmarks/common/benchmark_presentation.py

```diff
@@ -229,21 +365,106 @@ def categorize_benchmarks_into_tables(benchmarks: Dict[
 
   tables = []
   if regressed:
-    tables.append(md.header("Regressed Benchmarks 🚩", 3))
+    tables.append(md.header("Regressed Latencies 🚩", 3))
     tables.append(_sort_benchmarks_and_get_table(regressed, size_cut))
   if improved:
-    tables.append(md.header("Improved Benchmarks 🎉", 3))
+    tables.append(md.header("Improved Latencies 🎉", 3))
     tables.append(_sort_benchmarks_and_get_table(improved, size_cut))
   # If we want to abbreviate, similar results won't be interesting.
   if similar and size_cut is None:
-    tables.append(md.header("Similar Benchmarks", 3))
+    tables.append(md.header("Similar Latencies", 3))
     tables.append(_sort_benchmarks_and_get_table(similar, size_cut))
   if raw:
-    tables.append(md.header("Raw Benchmarks", 3))
+    tables.append(md.header("Raw Latencies", 3))
     raw_list = [(_make_series_link(k), v.mean_time, v.median_time,
                  v.stddev_time) for k, v in raw.items()]
     tables.append(
         _add_header_and_get_markdown_table(BENCHMARK_RESULTS_HEADERS,
                                            raw_list,
                                            size_cut=size_cut))
   return "\n\n".join(tables)
+
+
+def _sort_metrics_objects_and_get_table(metrics_objs: Dict[str, T],
+                                        mapper: MetricsToTableMapper[T],
+                                        headers: Sequence[str],
+                                        size_cut: Optional[int] = None) -> str:
+  """Sorts all metrics objects according to the improvement/regression ratio and
+    returns a markdown table for it.
+
+    Args:
+      metrics_objs: map of (name, metrics object).
+      mapper: MetricsToTableMapper for metrics_objs.
+      headers: list of table headers.
+      size_cut: If not None, only show the top N results for each table.
+  """
+  sorted_rows = []
+  for name, metrics_obj in metrics_objs.items():
+    current, base = mapper.get_current_and_base_value(metrics_obj)
+    if base is None:
+      raise AssertionError("Base can't be None for sorting.")
```

Comment:
Is this really an assertion? Is it caused by a programmer error. If so, this should at least document the constraint (all `metric_objs` must contain base values)